tests/utilities/test_utilities.py

```diff
@@ -361,6 +362,35 @@ def test_s_enhance_3D_no_obs(s_enhance):
                                                               f].mean())
 
 
+def test_t_coarsen():
+    """Test temporal coarsening of 5D array"""
+    t_enhance = 4
+    hr_shape = (3, 10, 10, 48, 2)
+    arr = np.random.uniform(-1, 1, )
```

Comment:
looks like you have a leftover here @grantbuster

Reply:
dang and just as the tests were almost done! haha no worries, i'll fix